backend/src/shared/error/locale.en.ts

```diff
@@ -17,6 +17,7 @@ const ERROR_EN: Record<ErrorCode, string> = {
   PAE013: "You cannot view this document",
   PAE014: "Wrong document's passcode",
   PAE015: "Cannot unshare (Wrong document's passcode)",
+  PAE0077: "Not a premium user",
```

Comment:
@tranqhuy46 K nghiêm trọng lắm, nh mà ông update cái error mes thành vầy giúp tui vs:
"You are using a free account.\n
Please upgrade your account for more premium features"

Reply:
roi nha Vu

Reply:
merge thaqnrg tiến